ruby_programming/testing_with_rspec/introduction_to_rspec.md

```diff
@@ -0,0 +1,236 @@
+# Introduction to RSpec
+
+## Introduction
+In the previous lesson, we established the utility of [test-driven development](https://en.wikipedia.org/wiki/Test-driven_development) (TDD) in maintaining your code and sanity. In this lesson, we'll introduce you to your new best friend, the [RSpec](http://rspec.info/) testing framework. RSpec is one of the most popular testing frameworks, having been downloaded over [100 million times](https://rubygems.org/gems/rspec), as of this writing, and having been ported for use in [Rails testing](https://rubygems.org/gems/rspec-rails).
```

Comment:
"as of the time of this writing" maybe?

Reply:
Or just omit it.

`RSpec .. having been downloaded over 100 million times.`

Reply:
When making a concrete observation at a point in time that is likely to change in the future, I think it's proper to allude to the time of writing.

Reply:
@CouchofTomato Changed to "It's" in the second sentence, to maintain flow with the subordinate clauses.